VideoWeb/VideoWeb/ClientApp/src/app/shared/error-camera-microphone/error-camera-microphone.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { Logger } from 'src/app/services/logging/logger-base';
+import { PageTrackerService } from 'src/app/services/page-tracker.service';
+import { SessionStorage } from 'src/app/services/session-storage';
+
+@Component({
+    selector: 'app-error-camera-microphone',
+    templateUrl: './error-camera-microphone.component.html'
+})
+export class ErrorCameraMicrophoneComponent implements OnInit {
+    deviceIsInUse = false;
+
+    private readonly loggerPrefix = '[ErrorCameraMicrophonePage] -';
+    readonly ERROR_CAMERA_MIC_MESSAGE_KEY = 'vh.error.camera.mic.message';
```

Comment:
Rather than creating a session storage object here to read, the error service should maintain the responsibility for saving and returning the error message rather than introducing this logic here

Reply:
It's done in the same way like it done for error.component. Just only I saved in storage flag indicated type of message on UI. And the service error maintain the responsibility for the type of error message.